Lesson_4/user-portait.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener('DOMContentLoaded', () => {
+   const userYearOfBirth = Number(prompt('Введіть Ваш рік народження'));
+   const userCity = prompt('В якому місті Ви живете?').toLowerCase();
```

Comment:
i think it will be better to use on check step, to avoid redundant replacement after

Reply:
add check